api/views/privaterelay.py

```diff
@@ -330,7 +330,7 @@ def terms_accepted_user(request):
     except SocialAccount.DoesNotExist:
         # User does not exist, create a new Relay user
         fxa_profile_resp = requests.get(
-            FXA_PROFILE_URL, headers={"Authorization": f"Bearer {token}"}
+            FXA_PROFILE_URL, headers={"Authorization": f"Bearer {token}"}, timeout=1
```

Comment:
This could be the same FxA setting

Reply:
Updated to use `settings.FXA_REQUESTS_TIMEOUT_SECONDS`